### PR TITLE
Add A/A generic experiment

### PR DESCRIPTION
**Pull request by marcosholgado**
**Asana Task/Github Issue:**
https://app.asana.com/0/1125189844152671/1208650074052221/f
## Description
This PR enables a generic A/A testing on Android for the new A/B/N framework. It also adds the cohort object to the TS schema.

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

**Comment by marcosholgado** [comment]
@sammacbeth can you have a look at this PR? I've added directly the A/A in the override because we don't really want this feature to exist forever so I'll just delete afterwards from there. There's also the TS validation schema changes here.

File: schema/feature.d.ts [hunk 1]
```diff
@@ -18,6 +23,7 @@ type SubFeature<VersionType> = {
     targets?: {
         variantKey?: string;
```
Comment (sammacbeth) [inline schema/feature.d.ts:24]: Should we also flesh out the additional supported targets mentioned in https://app.asana.com/0/481882893211075/1208204385124443/f?

Comment (marcosholgado) [reply schema/feature.d.ts:24]: Yeah, good shout. Done.

**Review by sammacbeth — APPROVED** [review]

**Comment by marcosholgado** [comment]
FYI will merge tomorrow morning just to be around to validate the changes in case I need to disable quick.

**Pull request merged**